src/bp-document/bp-document-filters.php

```diff
@@ -261,6 +261,20 @@ function bp_document_update_activity_document_meta( $content, $user_id, $activit
 		return false;
 	}
 
+	// Add in description in attachment when only one document uploaded.
+	if ( is_array( $_POST['document'] ) && 1 === count( $_POST['document'] ) ) {
+		if ( '' === trim( $content ) && ! empty( $activity_id ) ) {
+			$get_content = new BP_Activity_Activity( $activity_id );
+			$content = $get_content->content;
+		}
```

Comment:
@chetansatasiya we don't need to get content from the activity here. we already get the content in function arguments `( $content, $user_id, $activity_id )` if in there `$content` is empty we dont go into the condition and add attachment content.

Reply:
@mehulkaklotar DONE 👍